api/handlers.go

```diff
@@ -0,0 +1,57 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/SkynetLabs/malware-scanner/database"
+	"github.com/julienschmidt/httprouter"
+	"gitlab.com/NebulousLabs/errors"
+	skyapi "gitlab.com/SkynetLabs/skyd/node/api"
+)
+
+// healthGET returns the status of the service
+func (api *API) healthGET(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	status := struct {
+		DBAlive     bool `json:"dbAlive"`
+		ClamAVAlive bool `json:"clam_av_alive"`
+	}{}
+	err := api.staticClamav.Ping()
+	status.ClamAVAlive = err == nil
+	err = api.staticDB.Ping(r.Context())
+	status.DBAlive = err == nil
+	skyapi.WriteJSON(w, status)
+}
+
+// scanPOST adds a new skylink to the scanning queue. If the skylink is already
+// in the queue we respond with 200 OK but we don't add it again.
+func (api *API) scanPOST(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	skylink, err := parseSkylink(ps.ByName("skylink"), api.staticClamav.PreferredPortal())
```

Comment:
Follow up: should we add v1 validation here? Should be able to import that helper from skyd. Then it puts the requirement on the caller to only send v1 skylinks to the scanner.

Reply:
Do we want that limitation? Keep in mind that the called will typically be nginx and putting the onus on Karol to do skylink resolution in lua will be burdensome. I am already resolving all V2 skylinks to their respective V1s.

Reply:
F/U issue for discussion: https://github.com/SkynetLabs/malware-scanner/issues/5